tests/snmpsim/equallogic_1.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|User input
```

Comment:
Is this genuinely the sysDescr of the device?

Reply:
This was copied from equallogic.snmprec. On my device, it is an empty string.

Reply:
Ok, then just remove `User input`. It should represent exactly what you see.

Reply:
It's dependent on what the user inputs in the configuration interface, this is likely the reason that the person who created equallogic.snmprec used "User input" for the string. I chose to keep the description string empty because that fits my use case.

If you'd like, I can create it as an empty string; should I change equallogic.snmprec too?

Reply:
Leave equallogic.snmprec but set yours to blank.